test/shared/utils.py

```diff
@@ -36,17 +40,23 @@ def cibuildwheel_run(project_path, env=None, add_env=None):
 
     if add_env is not None:
         env.update(add_env)
+    
+    if output_dir is None:
+        output_dir = "wheelhouse"
 
     subprocess.check_call(
-        [sys.executable, '-m', 'cibuildwheel', project_path],
-        env=env,
+        [sys.executable, '-m', 'cibuildwheel', "--output-dir", output_dir, project_path],
+        env=env
     )
 
 
-def expected_wheels(package_name, package_version):
+def expected_wheels(package_name, package_version, add_env=None):
```

Comment:
Pffff, isn't there a better way? Can't we somehow pass `**kwargs` and just `tag_version = kwargs.get('macosx_tag_version')` and `'{package_name}-{package_version}-cp27-cp27m-macosx_' + (tag_version or '10_6') + '_intel.whl'`, or something like that?

I don't feel like `expected_wheels` should deal with environment variables.

Reply:
I do this with enviroment because I think aboutt fileterin result base on `CIBW_BUILD` and `CIBW_SKIP`, but not sure if it is proper approach.

Reply:
Hmmm, I don't really like that. Íf it needs to be filtered there (and I'm not really convinced of that either), we should at least pass a `BuildSelector` object.

Reply:
Ok. I will wait with any changes in code till the decision about which test PR is accepted.

Reply:
Sure, no rush!

Reply:
(It's also a minor detail, compared to all the work you have done in this PR, ofc, but since there's still time to fix things, ...)